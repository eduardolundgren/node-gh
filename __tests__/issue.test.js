/**
 * Â© 2013 Liferay, Inc. <https://liferay.com> and Node GH contributors
 * (see file: CONTRIBUTORS)
 * SPDX-License-Identifier: BSD-3-Clause
 */

const { runCmd } = require('./testUtils')

describe('E2E: Issues Module Test', () => {
    it('List Issues `gh is`', done => {
        expect(runCmd('bin/gh.js is')).toMatchSnapshot()
        done()
    })

    it('Comment on Issuess `gh is 1 -c "test"`', done => {
        expect(runCmd(`bin/gh.js is 1 -c "test"`)).toMatchSnapshot()
        done()
    })
})
