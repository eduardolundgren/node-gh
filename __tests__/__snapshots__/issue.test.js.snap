// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`E2E: Issues Module Test Comment on Issuess \`gh is 1 -c "test"\` 1`] = `
"Adding comment on issue #1
https://github.com/protoEvangelion/gh/issues/1#issuecomment-000000001
"
`;

exports[`E2E: Issues Module Test List Issues \`gh is\` 1`] = `
"Listing open issues on protoevangelion/gh
#43 testGo @protoEvangelion (a year ago)

#42 test2 @protoEvangelion (a year ago)

#38 Node GH rocks! @protoEvangelion (a year ago)
"
`;
